File: Array/print_first_repeat_number_in_array.py
N= int(input())
arr = list(map(int,input().split()))
count=0
for i in range(0,N):
    for j in range(i+1,N):
        if(arr[i]==arr[j]):
            count=count+1
    if(count==1):
        break
if(count==1):
    print(arr[i])
else:
    print("unique")

File: Javascript/codekata/array/print_first_repeat_number_in_array.js
const inp=readline.createInterface({
  input:process.stdin
});
const m=[];
inp.on("line",(data)=>{
  m.push(data);
});
inp.on("close",()=>{
  var A=m[0].split(" ");
  var B=m[1].split(" ");
  var count=0;
  for(var i=0;i<B.length;i++)
  {
    for(var j=i+1;j<B.length;j++)
    {
      if(B[i]==B[j])
      {
        count=count+1
        break
      }
    }
    if(count==1)
    {
      break
    }
  }
  if(count==1){
    console.log(B[i]);
  }
  else
  {
    console.log("unique");
  }
});
